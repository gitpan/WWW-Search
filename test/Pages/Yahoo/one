http://search.yahoo.com/search?b=0&h=s&n=68&p=LSAM&
