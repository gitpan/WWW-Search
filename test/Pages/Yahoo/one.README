This set of test-result pages was created on Wed Sep 29 10:34:22 1999
