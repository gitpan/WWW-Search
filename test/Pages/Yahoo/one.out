http://www.isi.edu/~johnh/SOFTWARE/APACHE/index.html
http://www.lsamr.com
http://www.isi.edu/lsam/publications/http-perf/index.html
http://www.isi.edu/lsam/publications/phttp_tcp_interactions/index.html
http://www.lsamr.com
http://www.fis.puc-rio.br/lsam.d/hall_eng.html
